scrapi/harvesters/bhl.py

```diff
@@ -12,7 +12,6 @@ def institution_name_parser(names):
     for inst in names:
         contributor = {
             'name': inst,
-            'email': '',
             'sameAs': []
```

Comment:
No, required fields are not looked into (for now). Empty sameAs is fine for validation and doesn't hurt search though, so I figured I wouldn't take it out (an empty list is different than an empty string- an empty list is basically a null value for lists (it contains no information), while an empty string is actually a value inside string that could have meaning)